web/src/features/fireBehaviourCalculator/FireBehaviourCalculatorPage.tsx

```diff
@@ -103,6 +105,15 @@ export const FireBehaviourCalculator: React.FunctionComponent = () => {
     setRowId(lastId + 1)
   }, [location]) // eslint-disable-line react-hooks/exhaustive-deps
 
+  const disableCalculateButton = some(
```

Comment:
Maybe throw grass cure check in here on grass types?

Reply:
Grass cure is optional on the backend request model.

Reply:
Totally misunderstood your comment. Will do.

Reply:
Unfortunately there's no way around marking grass cure as optional, as it's only required in combination with grass fuel types.
But your right contract wise. We can have the back-end return an error for that entry, instead of just breaking. That's probably even be a better way of doing it. We did use to have that check on the front end, and I think the ideal would probably be addressing it on both ends.
However we do it, I think we should have it as part of this ticket.